src/index.js

```diff
@@ -4,6 +4,22 @@ import bindActionCreators from './utils/bindActionCreators'
 import applyMiddleware from './utils/applyMiddleware'
 import compose from './utils/compose'
 
+/*
+* create a function so that we can check if the function name has been altered by minification
+* if the function has been minified and NODE_ENV !== 'production', warn the user
+*/
+function isCrushed() {}
+
+if (isCrushed.name !== 'isCrushed' && process.env.NODE_ENV !== 'production') {
```

Comment:
That comes from the Node universe, where `process.env.NODE_ENV` is defined by default. `process` is a global in Node. It is used in many libraries (React included) to define where development code is. Our module bundler (and most common setups) replaces the text `process.env.NODE_ENV` with `"production"`, which gets eliminated by an optimizer as dead code. 

So, it would be a good idea to emulate the same behavior, even if you don't like it. There are some Babel plugins to make this easier (letting you use a global __DEV__). You're probably getting development builds of many other things for the same reason.